compiler/src/dmd/file_manager.d

```diff
@@ -23,6 +28,7 @@ import dmd.location;
 
 enum package_d  = "package." ~ mars_ext;
 enum package_di = "package." ~ hdr_ext;
+enum ext_sar    = "sar";        // file extension for source archive
```

Comment:
Why doesn't this begin with `.d`?

Reply:
??

Reply:
.dar files are already Disk ARchive files. .sar files are a variety of formats.

Possible conflation with other formats is why the header has a magic number and numerous checks for goodness.